spec/kuma_plugins/liquid/tags/inc_spec.rb

```diff
@@ -0,0 +1,70 @@
+RSpec.describe Jekyll::KumaPlugins::Liquid::Tags::Inc do
+  let(:version) { '2.9.1' }
+  let(:site) { Jekyll::Site.new(Jekyll.configuration({ mesh_namespace: "kuma-demo" })) }
+  # If we ever upgrade jekyll-generator-single-source we will have to change below to:
+  # let(:release) { Jekyll::GeneratorSingleSource::Product::Release.new({ 'release' => version }) }
+  # let(:page) { { 'release' => release.to_liquid } }
+  let(:page) { { 'version' => version } }
+  let(:environment) { { 'page' => page } }
+  let(:registers) { { page: page, site: site } }
+  let(:liquid_context) { Liquid::Context.new(environment, {}, registers) }
+
+  shared_examples 'renders inc tag' do |golden_file, params = []|
+    tag_params = ["some_value", *params].join(' ')
+
+    it "renders correctly for parameters: #{tag_params}" do
+      template = "{% inc #{tag_params} %}"
+      processed = OpenStruct.new(content: template)
+      Jekyll::Hooks.trigger(:pages, :pre_render, processed)
+
+      output = Liquid::Template.parse(processed.content).render(liquid_context)
+      GoldenFileManager.assert_output(output, golden_file, include_header: false)
+    end
+  end
+
+  describe 'rendering tests' do
+    [
+      { golden_file: 'spec/fixtures/inc.golden' },
+      {
+        golden_file: 'spec/fixtures/inc-if-version.golden',
+        parameters: ['if_version=gte:2.9.x'],
```

Comment:
I was thinking more about having the whole `if_version` block with `inc` so that the whole list get's rendered so we see the correct values being rendered 🤔

Reply:
wdyt now?